nixos/modules/misc/ids.nix

```diff
@@ -666,6 +667,7 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
+      audiobookshelf = 327;
```

Comment:
No need for static uid, those are legacy.

Just declare `users.groups.audiobookshelf` without it.